src/main/java/PlayerFactory.java

```diff
@@ -0,0 +1,19 @@
+public class PlayerFactory {
+    private final Display display;
+    private final InputValidator inputValidator;
+    
+    public PlayerFactory
+            (Display display, InputValidator inputValidator) {
+        this.display = display;
+        this.inputValidator = inputValidator;
+    }
+    
+    public Player createPlayer(String playerType, String mark) {
+        
+        if (playerType.equals("h")) {
```

Comment:
Could this be more descriptive?

Reply:
Good point, I'll change this. Thanks, Ashley